Makefile

```diff
@@ -85,8 +85,8 @@ docker-build: load-design-system-templates  ## Build Docker container
 	docker compose pull
 	docker compose build
 
-.PHONY: docker-start
-docker-start:  ## Start Docker containers
+.PHONY: docker-up
```

Comment:
You haven't updated the README to reflect this change: `README line 95`. It still says `make docker-start`, it should be `make docker-up` now due to this change.

Reply:
I propose using `compose` instead of `docker` in our command names. Starting with an action word makes commands clearer and more intuitive. For instance, `make compose-up` / `make compose-up-dev` is more descriptive than `make docker-up` / `make docker-dev-up`. The term `docker-up` could be misinterpreted as starting the Docker daemon itself, whereas compose better conveys the intent of using Docker Compose. In past projects, we’ve used compose, and developers found it clearer and more meaningful.

If retaining the term docker is important, we could consider `docker-compose-up` for clarity, though shorter commands are generally preferable if they remain meaningful. See what the team thinks and go with the consensus.

---

Also, I recommend ensuring commands follow consistent prefixes. This approach improves shell autocomplete functionality by grouping related commands, aligning with common patterns for shell commands. For example, using `makemigrations` and `makemigrations-check` is more intuitive than `makemigrations` and `check-makemigrations`. This patterns allows developers to easily discover and understand available commands.

Lastly, I suggest dropping `make` from command names like `makemigrations`. Typing `make makemigrations` feels redundant and less natural. Instead, aliases like `make migrations` and `make migrate` are more straightforward and intuitive. While the command names don’t need to perfectly mirror the underlying operations, they should be clear and easy to use. Again, see what the team prefers and go with the consensus.

Reply:
I agree with the idea to replace `docker` with `compose` in the Make commands where appropriate.

I think that it makes more sense to have the prefix of the dev commands as `compose-dev...` so in this example it'd be `make compose-dev-up` rather than `make compose-up-dev`. 
This would mean that the shell autocomplete would be performed for all the `compose-dev-...` commands and print all the available dev commands rather than run for `compose-up-...` which would only print the dev and non-dev variations of the command.

Reply:
Re. the last paragraph, personally I'd still stick to using `makemigrations` rather than `migrations` as name of the make command.

Semantically the `make` command is completely separate to the "make" in `makemigations` - it doesn't replace it and hence we shouldn't try to trade clarity for conciseness.

Secondly, in your comment you highlight that using verbs in the make commands makes them more intuitive and easier to read. I think that's the most convincing argument for me here.

Thirdly IMO `migrations` and `migrate` would be dangerously close to each other - at first sight you're not able to tell what each one is actually doing. 

Also, for all the other all the other Make recipes we use exactly the same naming to `django-admin` commands. I think it's beneficial to keep it consistent.

I suggest that the reviewers react to this comment to show their preference:
👍 - rename it to `make migrate`
👎  - keep it as `make makemigrations`

Reply:
> I agree with the idea to replace `docker` with `compose` in the Make commands where appropriate.
> 
> I think that it makes more sense to have the prefix of the dev commands as `compose-dev...` so in this example it'd be `make compose-dev-up` rather than `make compose-up-dev`. This would mean that the shell autocomplete would be performed for all the `compose-dev-...` commands and print all the available dev commands rather than run for `compose-up-...` which would only print the dev and non-dev variations of the command.

LGTM.

Reply:
> Re. the last paragraph, personally I'd still stick to using `makemigrations` rather than `migrations` as name of the make command.
> 
> Semantically the `make` command is completely separate to the "make" in `makemigations` - it doesn't replace it and hence we shouldn't try to trade clarity for conciseness.
> 
> Secondly, in your comment you highlight that using verbs in the make commands makes them more intuitive and easier to read. I think that's the most convincing argument for me here.
> 
> Thirdly IMO `migrations` and `migrate` would be dangerously close to each other - at first sight you're not able to tell what each one is actually doing.
> 
> Also, for all the other all the other Make recipes we use exactly the same naming to `django-admin` commands. I think it's beneficial to keep it consistent.
> 
> I suggest that the reviewers react to this comment to show their preference: 👍 - rename it to `make migrate` 👎 - keep it as `make makemigrations`

Yup, I don't have a strong view, happy with either, the main ask is that, it is clear to devs so go with the consensus.

Reply:
Fixed the typo 👍